.github/workflows/ci.yaml

```diff
@@ -609,6 +609,7 @@ jobs:
           ACCEPT_EULA: "Y"
         ports:
           - 1433:1433
+        options: --name sqlserver17
```

Comment:
To run the `docker exec` command to create `no superuser`, we need to use the container name as an argument of the `docker exec` command.
So, I specify the `--name` option to set the container name explicitly.